README.md

```diff
@@ -84,3 +84,11 @@ python -m alf.bin.play --root_dir=LOG_DIR
   Python package gym-retro>=0.7.0 is required for this experiment and also a suitable `SuperMarioBros-Nes` rom should be obtained and imported (roms are not included in gym-retro). See [this doc](https://retro.readthedocs.io/en/latest/getting_started.html#importing-roms) on how to import roms.
 
   <img src="alf/examples/icm_super_mario_intrinsic_only.png" width = "300" height ="200" alt="super-mario-training-curve"/> <img src="alf/examples/icm_super_mario_intrinsic_only.gif" width = "300" height ="200" alt="super-mario-video"/>
+  
+
+### Merlin
+* [Room collect good objects](alf/examples/merlin_dmlab_collect_good_objects.gin). 
```

Comment:
This bullet should be called "Collect good objects in rooms"? (current syntax seems strange). Also please add a sentence briefly summarizing what algorithm (MERLINE in this case) solves what kind of problem, just like in the ICM case above.